doc/ref_impl/cntt-ri/chapters/chapter07.md

```diff
@@ -71,18 +72,67 @@ Details explaining:
 - Disposition, or next steps, based on test results (e.g. handling of failures, handoff to next tier with passing results)
 
 <a name="7.8"></a>
-## 7.8 CICD Tool Chain (use of, process, and scripts)]
+
+## 7.8 Development Validations
+
+CNTT RI jobs must verify all patches before merge as defined in the best open
+source practices (see
+[OpenStack Gates](https://docs.openstack.org/infra/system-config/devstack-gate.html)
+or [Functest Gates](https://build.opnfv.org/ci/view/functest/job/functest-hunter-gate/142/))
+to achieve the stability needed by CNTT. Then the deployment validations
+previously detailed must be also applied for every patch proposed in RI and
+all changes published in [Gerrit](https://gerrit.opnfv.org/) must be
+automatically voted -1 by Jenkins in case of failures.
+
+Only the following test cases must pass as a temporarily bypass in
+[RI gates](https://build.opnfv.org/ci/view/airship/job/airship-latest-gate/) to
+allow merging the next patches. At the time writing, CNTT RI is not compliant
+with [RA1 Core OpenStack Services APIs]({{ "/doc/ref_arch/openstack/chapters/chapter05.html" | relative_url }})
+and then a part of the containers selected in [RC1 TC Requirements]({{ "/doc/ref_cert/lfn/chapters/chapter09.html" | relative_url }})
```

Comment:
Relative URL paths do not appear to be working.  Can you resolve?

Reply:
It works. It's postprocessed automatically and changed to the right path. That's part of the best practices to manage url prefix in Jekyl (https://cntt-n.github.io/CNTT)

Reply:
If there's a way to preview, let me know.  I can preview with markdown, using either StackEdit or GitHub.  Not able to preview / resolve your URL.  Regardless, I'll approve, and we can resolve broken URLs in subsequent patches.  The content is good.